en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md

```diff
@@ -0,0 +1,51 @@
+# Migrating from WSO2 EI 7.0.0 to WSO2 EI 7.1.0
+
+### Set up the migration
+
+-	Make a backup of the existing database used by the current EI 7.0.0 deployment. This backup is necessary in case the migration causes any issues in the existing database.
+-	Download and install EI 7.1.0 in your environment. The home directory of your installation will be referred to as `<MI_HOME>` from hereon.
+	-	[On a VM](../../../setup/installation/install_in_vm)
+	-	[On Docker](../../../setup/installation/run_in_docker)
+	-	[On Kubernetes](../../../setup/installation/run_in_kubernetes)
+-	You can [use WSO2 Update Manager](https://docs.wso2.com/display/updates/) to get the latest available updates.
+
+	!!! Info
+		Note that you need a valid [WSO2 subscription](https://wso2.com/subscription) to use updates in a production environment.
+
+### Migrating databases
```

Comment:
We didn't support databases in EI 7.0.0 did we? I think we can remove this section.
@Nirothipan - could you also confirm?

Reply:
+1 to make as 'Migrating user stores'  .. we can say just plug the same here and point to user store docs.

Reply:
Done.